### ISSUE TITLE
Live stream is seriously broken

### ISSUE THREAD

**Issue by solbu** [Cat5TV/plugin.video.category5#17]
I am just about to give up on the entire live stream. :-(
I cannot get it to play beyond a few seconds, before it starts to buffer, and it is only the cat5 stream this happens on. Every other live stream I play in kodi play without any issues.

I have attached a kodi debug log, in case it makes sense to anyone.

[kodi-log.txt](https://github.com/Cat5TV/plugin.video.category5/files/2489721/kodi-log.txt)

**Comment by Cat5TV** [comment]
Thanks for letting us know. Dang. Are you available after-hours for testing sometime soon (before next Wednesday, I hope)? I could setup a test feed and make changes to the codecs, with you testing to see if they improve things.

**Comment by solbu** [comment]
Sure, I can help test. We can discuss time on email, as we have timesone differences. :-)
The show starts at 1am here where I live. It is currently 2:31 am, which is 8:31pm in your timezone.

**Comment by solbu** [comment]
Just for reference, before you had to reinstall the windows system at your end, the stream was excellent. It worked flawless.

**Comment by Cat5TV** [comment]
That's great to know - thank you. Upon reinstalling Wirecast, we were regressed to "default" codec settings. I can go back over everything and revert them to our old settings. Can restore them from backup. Sorry it's been problematic.
Yes, please send me an email (not tonight - go get some rest) and let's coordinate a test time so we can ensure you can watch flawlessly next week  :) Thanks so much!

**Comment by Cat5TV** [comment]
I have re-created our video encoding profile from before the Wirecast upgrade. Since the settings match from the time you say it worked flawlessly, I imagine this should fix it for you. Please let me know if you are able to test off-air (I can just stream a previous episode), otherwise, please let me know during Wednesday's live show if it is fixed for you. Thanks!

**Comment by Cat5TV** [comment]
The recorded version of the modified live stream is up - please test and let me know if it is working now. Thanks!

**Comment by solbu** [comment]
And … I still have issues, but it's different this time.
But, your re-creation at least changed it for the better.

I streamed for about 40 minutes, and just when you went live it craped out, and I had to restart. Then it stuttered for a few seconds, and went smootly for abuot 2 minutes 30 sec. And the same issue as before came back. It then works for 30 seconds, and crawls to a halt like before. When I restart the stream, it works for 2 and a half minute, and cralws to a halt.

Something really weird is happening.

I am running Kodi 17.6 on Mageia.
«iftop» reports that I get aprox 2 Mbit stream from percy (your server).
